test/unittest/statistics/dds/StatisticsDomainParticipantMockTests.cpp

```diff
@@ -42,11 +42,7 @@ class StatisticsDomainParticipantMockTests : public ::testing::Test
     void helper_block_for_at_least_entries(
             uint32_t amount)
     {
-        std::unique_lock<std::mutex> lck(*mutex_);
```

Comment:
I think this mutex is only used here. Remove it.

Reply:
done